src/portable/link.rs

```diff
@@ -422,15 +423,15 @@ async fn prompt_conn_params(
 pub fn print_warning(name: &str, project_dirs: &[PathBuf]) {
     project::print_instance_in_use_warning(name, project_dirs);
     eprintln!("If you really want to unlink the instance, run:");
-    eprintln!("  edgedb instance unlink -I {:?} --force", name);
+    eprintln!("  {BRANDING_CLI_CMD} instance unlink -I {:?} --force", name);
 }
 
 pub fn unlink(options: &Unlink) -> anyhow::Result<()> {
     let name = match instance_arg(&options.name, &options.instance)? {
         InstanceName::Local(name) => name,
         inst_name => {
             return Err(anyhow::anyhow!(
-                "cannot unlink cloud instance {}.",
```

Comment:
I feel like I'd prefer not to change all of the unqualified "cloud" references to "Gel Cloud". But also I don't feel strongly about it, especially because I'm sure preparing this PR was already enough of a nightmare without having to undo some of it.